src/Sarif.Viewer.VisualStudio/ErrorList/ISarifErrorListEventSelectionService.cs

```diff
@@ -23,6 +24,14 @@ internal interface ISarifErrorListEventSelectionService
         /// </remarks>
         SarifErrorListItem SelectedItem { get; set; }
```

Comment:
>SelectedItem [](start = 27, length = 12)

do we need this?

Reply:
It is used in other places like TextMarker/Tag, it has different logic e.g. when multiple item selected, it will be set to null to avoid drawing multiple TextMarker/Tag.
I think we cannot remove it.

---
In reply to: [563568803](https://github.com/microsoft/sarif-visualstudio-extension/pull/353#discussion_r563568803) [](ancestors = 563568803)